### PR TITLE
Update docker image with data.json

### PR DESCRIPTION
**Pull request by ruflin**
* Add docs for labels and tags

### CODE REVIEW

File: metricbeat/module/docker/container/_meta/fields.yml [hunk 1]
```diff
@@ -40,3 +40,13 @@
...
+    - name: labels
+      type: dict
+      dict-type: keyword
+      description: >
+        Image labels.
+    - name: tags
+      type: list
+      dict-type: keyword
+      description: >
+        Image labels.
```
Comment (douaejeouit) [inline metricbeat/module/docker/container/_meta/fields.yml]: @ruflin Don't you think that the tags description need to be updated?

Comment (ruflin) [inline metricbeat/module/docker/container/_meta/fields.yml]: Definitively. Fixed. Thanks for spotting.

Comment (andrewkroh) [inline metricbeat/module/docker/container/_meta/fields.yml]: Does this file need re-generated? I think it's supposed to say "Image tags"?

Comment (ruflin) [inline metricbeat/module/docker/container/_meta/fields.yml]: No, my mistake. I updated the wrong file and then run `make update` :-( New version pushed.

**Pull request merged**